examples/org.eclipse.xtext.xtext.ui.examples/contents/org.eclipse.xtext.example.arithmetics.ide/.settings/org.eclipse.core.resources.prefs

```diff
@@ -1,2 +1,2 @@
 eclipse.preferences.version=1
-line.separator=\n
```

Comment:
We should configure the line endings per project, otherwise we will have changes in our dev workspace when checked out on windows